### ISSUE TITLE
main_bayesian_regression_hetero.py

### ISSUE THREAD

**Issue by kikyo97** [Neronjust2017/BayesianCNN#1]
大佬 @Neronjust2017 ，在main_bayesian_regression_hetero.py这个文件里
        outputs = torch.mean(outputs, dim=2)
        if outputs.shape[1] == 1:
            outputs = outputs.reshape([outputs.shape[0]])
        samples = outputs[:,:1].reshape([outputs.shape[0]])
        noises = outputs[:,1:].reshape([outputs.shape[0]])
outputs 经过第一行不是已经只有两个维度了么？
if语句之后就只剩下0这个维度，下面的samples和noises里outputs为什么会有第二个维度呢？
